homeassistant/components/apsystems/coordinator.py

```diff
@@ -38,6 +38,7 @@ def __init__(self, hass: HomeAssistant, api: APsystemsEZ1M) -> None:
             update_interval=timedelta(seconds=12),
         )
         self.api = api
+        self.device_version = None
```

Comment:
Instead, overwrite the type on class level. We know that the variable will have a value, because if the coordinator is unable to pass `_async_setup`, we are not going to set up